src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java

```diff
@@ -48,6 +49,7 @@ public Boolean apply(WebDriver from) {
             return elementsAttributeValue.contains(value);
           }
         } catch (StaleElementReferenceException e) {
+          LOG.info("STALE REFERENCE", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "STALE REFERENCE" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)